dist/pythonlibs/testrunner/__init__.py

```diff
@@ -22,10 +22,12 @@
 TIMEOUT = int(os.environ.get('RIOT_TEST_TIMEOUT') or 10)
 
 
-def run(testfunc, timeout=TIMEOUT, echo=True, traceback=False):
+def run(testfunc, timeout=TIMEOUT, echo=True, traceback=False, sync=False):
```

Comment:
Have you considered to make sync default to True? Wouldn't that reduce the number of files you have to change for this PR?

Reply:
Sure, I didn't initially because I wasn't sure if the PR would need to be split and introduce changes slowly, in which case a default `false` was better.

I can remove them at the end, for now I rather keep it help me see easily which tests I have changed and which not :)

Reply:
> I can remove them at the end, for now I rather keep it help me see easily which tests I have changed and which not :)

By the end I mean after squashing :)

Reply:
Maybe it is time now, if all tests are not regressing, please squash and update the with default `sync=True`

Reply:
I'll test the squash version then push.